app/assets/stylesheets/practices.scss

```diff
@@ -1,3 +1,55 @@
-// Place all the styles related to the Practices controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+h3 {
+  font-weight: 900;
+  margin: 10px;
+}
+
+.profile {
+  background-color: #fff;
+  border-bottom: 1px solid #ddd;
+  margin: none;
+  padding: 10px;
+}
+
+.practices-list {
+  list-style-type: none;
+  margin: 20px 5px;
+
+  .practice {
+    background-color: #fff;
+    border: 1px solid #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.